### PR TITLE
qhc-1388-bug-qpqbloxplay-with-0-duration

### PR DESCRIPTION
**Pull request by flavie-lebars**
Fixed a bug where `qp.qblox.play` with `wait_time=0` was treated as no `wait_time` provided, producing incorrect Q1ASM. The wait time is now correctly clamped to the minimum valid value of 4 ns.

### CODE REVIEW

**Review by fedonman — APPROVED** [review]
LGTM 💯

File: src/qililab/qprogram/qblox_compiler.py [hunk 1]
```diff
@@ -1680,7 +1680,7 @@ def _handle_play(self, element: Play):
         if waveform_variables:
             logger.error("Variables in waveforms are not supported in Qblox.")
             return
-        if element.wait_time:
+        if element.wait_time is not None:
```
Comment (DZPM) [inline src/qililab/qprogram/qblox_compiler.py:1683]: Suggestion: adding a test where wait_time=None will make this more resilient

**Review by DZPM — APPROVED** [review]

File: src/qililab/qprogram/qblox_compiler.py (hunk 1 above)
Comment (flavie-lebars) [reply src/qililab/qprogram/qblox_compiler.py:1683]: I'll add it. Note there's already a test for wait_time=0 which covers the actual behavioral change. This one is just verifying that None is not None evaluates to False.
